addons/website/static/src/snippets/s_website_form/000.js

```diff
@@ -402,6 +410,20 @@ odoo.define('website.s_website_form', function (require) {
             });
         },
 
+        resetForm() {
```

Comment:
shouldn't it be in the private section? with docstring ?

Reply:
I am not sure about the private :thinking: Because this function simply adds things to remove in addition to the original reset (which is the JS one) so for me it should be public.
But yes I forgot the doc :grimacing: